.github/workflows/docs-deploy-firebase.patch.yml

```diff
@@ -18,7 +18,7 @@ on:
       - '.cargo/config.toml'
       - '**/clippy.toml'
       # workflow definitions
-      - '.github/workflows/docs.yml'
+      - '.github/workflows/docs-deploy-firebase.yml'
```

Comment:
When we change the workflow file name it also needs to change in the patch workflow.

I applied this change to the branch because I can't comment on unchanged files.